css/style.css

```diff
@@ -187,7 +187,7 @@ a {
 }
 
 #who .who-img {
-  background: url(./img/people.jpg) no-repeat center center/cover;
+  background: url(../img/people.jpg) no-repeat center center/cover;
```

Comment:
Who img incorrect file path has been solved. Previous commit was not successfull. Now this is finally fixed.